packages/ava/src/advisor/ruler/interface.ts

```diff
@@ -1,7 +1,12 @@
-import { analyzer } from '@antv/data-wizard';
+/**
```

Comment:
```suggestion
/*
```
this is a js multi-line comment for file, not jsdoc

and,the name of this file should be `types.ts`

Reply:
done